src/components/Card.tsx

```diff
@@ -4,7 +4,11 @@ import { urlForImage } from '~/lib/sanity.image'
 import { PostType } from '~/lib/sanity.interfaces'
 import { formatDate } from '~/utils'
 
-export default function Card({ post }: { post: PostType }) {
+interface CardProps {
```

Comment:
It is basically the same, but just different syntaxis but it should work in both cases